source/features/pull-request-timeline-filter.tsx

```diff
@@ -0,0 +1,261 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function sleep(s: number): Promise<any> {
+	await new Promise(resolve => setTimeout(resolve, s * 1000));
+}
```

Comment:
How should I fix lint issue here?

Reply:
We already use a package called `delay`

Reply:
But this part of the feature should be dropped as it was already rejected: https://github.com/sindresorhus/refined-github/issues/3814

Reply:
I removed this feature.

Reply:
Adding a comment to test this feature